### PR TITLE
[DOCS] Update Main Page for New Structure

### PR DESCRIPTION
**Pull request by shir22**
Also incorporate some previous comments, later to be implemented in monitoring docs.
Built version here:
https://drive.google.com/file/d/1uIPOy6avly-M2mAIGCLWSXj9C-l-l_Kw/view?usp=share_link

### CODE REVIEW

File: docs/source/getting-started/welcome.rst [hunk 1]
```diff
@@ -1,46 +1,86 @@
...
+    .. grid-item-card:: 🤓 General: Concepts & Guides 🤓
+        :link-type: ref
+        :link: general__index
+         
+        A comprehensive view of deepchecks concepts,
+        customizations, and core use cases.
+
+    .. grid-item-card:: 🔢 Tabular 🔢
+        :link-type: ref
+        :link: tabular__index
```
Comment (noamzbr) [inline docs/source/getting-started/welcome.rst:63]: I'm not sure I understand why here the links are tabular__index and not tabular_index, but trust that it works(:

File: docs/source/getting-started/welcome.rst [hunk 2]
```diff
@@ -140,4 +170,19 @@ Head over to the relevant documentation for more info:
...
+
+.. admonition:: Join Our Community 👋
+   :class: tip
+
+   In addition to perusing the documentation, feel free to:
+
+   - Ask questions on our `Slack Community <https://www.deepchecks.com/slack>`__,
+   - Post an issue or start a discussion on `Github Issues <https://github.com/deepchecks/deepchecks/issues>`__.
+
+   To support us, please give us a star ⭐️ on `Github <https://github.com/deepchecks/deepchecks>`__, it really means a lot for open source projects!
```
Comment (noamzbr) [inline docs/source/getting-started/welcome.rst:186]: Add link to contribution guide?

**Review by noamzbr — APPROVED** [review]
Maybe I can piggyback and ask to add the paper citation here?
https://github.com/deepchecks/deepchecks/issues/2464

File: docs/source/getting-started/welcome.rst (hunk 1 above)
Comment (shir22) [reply docs/source/getting-started/welcome.rst:63]: that's the name of the label :)
and also our convention: filepath to directory (or just nlp/tabular/cv), two underscores, and then two underscores and name of file

(that's why I also fixed in this PR one that wasn't like that)

File: docs/source/getting-started/welcome.rst (hunk 2 above)
Comment (shir22) [reply docs/source/getting-started/welcome.rst:186]: done

**Pull request merged**